ts/src/apex.ts

```diff
@@ -733,8 +735,6 @@ export default class apex extends Exchange {
         const symbol = this.safeSymbol (marketId, market);
         const last = this.safeString (ticker, 'lastPrice');
         const percentage = this.safeString (ticker, 'price24hPcnt');
-        const percent = Precise.stringMul (percentage, '100');
-        const open = Precise.stringDiv (last, Precise.stringMul ('1', percentage), 8);
```

Comment:
idk what was this `open` calculation. shortly, we should remove the derived calculations from implementation and `safeTicker` will do its job.